src/cmd/services/m3coordinator/ingest/write.go

```diff
@@ -94,18 +97,22 @@ type downsamplerAndWriter struct {
 	store       storage.Storage
 	downsampler downsample.Downsampler
 	workerPool  xsync.PooledWorkerPool
+
+	metricsDropped tally.Counter
```

Comment:
nit: Conventionally we tend to put this on a separate metrics struct.

i.e.
```go
type downsamplerAndWriter struct {
  // ... 
  metrics downsamplerAndWriterMetrics
}

type downsamplerAndWriterMetrics struct {
  dropped tally.Counter
}
```

This way when you inc a metric it's more readable, e.g.:
```go
w.metrics.dropped.Inc(1)
```